packages/design-system/src/components/N8nNotice/Notice.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+	<div :id="id" :class="classes" role="alert">
+	  <div class="notice-content">
+			<n8n-text size="small">
+				<slot>
+					<span
+						:class="expanded ? $style['expanded'] : $style['truncated']"
+						:id="`${id}-content`"
+						role="region"
+						v-html="renderedContent"
+					/>
+					<span v-if="canTruncate">
+						<a
+							role="button"
+						  :aria-controls="`${id}-content`"
+						  :aria-expanded="canTruncate && !expanded ? 'false' : 'true'"
+							@click="toggleExpanded"
+						>
+							{{ t(expanded ? 'notice.showLess' : 'notice.showMore') }}
+						</a>
+					</span>
+				</slot>
+			</n8n-text>
+	  </div>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import sanitizeHtml from 'sanitize-html';
+import N8nText from "../../components/N8nText";
+import Locale from "../../mixins/locale";
+import {uid} from "../../utils";
+
+const DEFAULT_TRUNCATION_MAX_LENGTH = 150;
+
+export default Vue.extend({
+	directives: {},
+	mixins: [
+	  Locale,
+	],
+	props: {
+		id: {
+			type: String,
+			default: () => uid('notice'),
+		},
+		type: {
+			type: String,
+			default: 'warning',
+		},
+		truncate: {
+			type: [Boolean, Number],
+			default: false,
```

Comment:
this prop is not clear.. might be clearer to pass in two props `truncate` and `truncateAt` (with a default value of 150)

Reply:
Good point! Changed to using `truncate` and `truncateAt`.